beacon-chain/core/blocks/withdrawals.go

```diff
@@ -196,26 +177,34 @@ func ProcessWithdrawals(st state.BeaconState, executionData interfaces.Execution
 			return nil, fmt.Errorf("unable to dequeue partial withdrawals from state: %w", err)
 		}
 	}
+	
+	if st.Version() >= version.EPBS {
+		if err := st.DequeuePartialWithdrawals(partialWithdrawalsCount); err != nil {
```

Comment:
`DequeuePartialWithdrawals` is a electra function no?

Reply:
We need to create new fn for this ig then coz I cant see any impl for epbs